demo/main/public/presets/zIndex.json

```diff
@@ -27,7 +27,7 @@
         "color": {
           "value": "#0000ff"
         },
-        "zIndex": -50
+        "zIndex": -50000
```

Comment:
This should be -5000, there is an additional 0

Reply:
Is it ok to leave out-of-bound values for testing in these examples?

Reply:
You can use tests for that, I prefer having good presets for testing visual effects

Reply:
Ok! I'm going to add a new commit with the z-index configuration values any second now.